README.md

```diff
@@ -1 +1,46 @@
-todo: write readme
\ No newline at end of file
+# Very Good WearOS App
+
+[![Very Good Ventures][logo_white]][very_good_ventures_link_dark]
+[![Very Good Ventures][logo_black]][very_good_ventures_link_light]
+
+Developed with 💙 by [Very Good Ventures][very_good_ventures_link] 🦄
+
+![coverage][coverage_badge]
+[![style: very good analysis][very_good_analysis_badge]][very_good_analysis_link]
+[![License: MIT][license_badge]][license_link]
+
+A Very Good Flutter Starter Project for WearOS devices created by the [Very Good Ventures Team][very_good_ventures_link].
+
+It includes the basic initial setup and some WearOS specific features to demonstrate how to take advantage of hardware resources.
+
+Generated by the [Very Good CLI][very_good_cli_link] 🤖
+
+--- 
+
+## Getting Started 🚀
+
+. Install [mason][mason_link]
+
+`dart pub global activate mason_cli`
+
+2. Add the brick
+
+   `mason add -g very_good_wear_app`
+
+3. Make a new WearOS app
+
+   `mason make very_good_wear_app`
+
+
```

Comment:
Should we include instructions on how to edit the app icon (it tends to be frequently be asked)?

Reply:
That is a fair request that also transcend to very good core. But we don't need to solve all the problems in the first go.